examples/cordova-sample/MyApp/package.json

```diff
@@ -23,15 +23,15 @@
     "plugins": {
       "cordova-plugin-whitelist": {},
       "cordova-annotated-plugin-android": {},
-      "cordova-plugin-purchases": {}
+      "cordova-plugin-add-swift-support": {}
     },
     "platforms": [
-      "ios",
```

Comment:
was this intentional?

Reply:
no lol, sorry I think I pushed some stuff that I shouldn't have 🤕